bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -62,7 +62,6 @@ class VideoProvider extends Component {
     super(props);
 
     this.state = {
-      sharedWebcam: false,
```

Comment:
Was there a specific reason why `sharedWebcam` is not in the state anymore?

Reply:
It had no use as a state (it's only an instance attribute to mark the provider as having a shared webcam or not for removing it in the unmount procedure), and sometimes it triggered a state update that would make the webcam be shared again if the negotiation procedure wasn't finished yet.